src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
+/* eslint-disable max-len */
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
```

Comment:
You could also use destructuring here:
```suggestion
export const MovieCard: React.FC<Props> = ({
  movie: {
    title,
    description,
    imgUrl,
    imdbUrl,
  },
}) => 
```